src/options.c

```diff
@@ -348,6 +348,11 @@ static void mi_error_default(int err) {
     abort();
   }
 #endif
+#if MI_XMALLOC
+  if (err==ENOMEM) { // abort on memory allocation fails in xmalloc mode
```

Comment:
Perhaps this should be `if (err==ENOMEM || err==EOVERFLOW)` as that catches all cases where `NULL` is returned.
The other `EAGAIN`, `EFAULT`, and `EINVAL` are only triggered at `free` or checks and can continue.

Reply:
I think yes. This would catch things like malloc(-1) :)